### PR TITLE
avoid double encoding for tm:re

### PR DESCRIPTION
**Pull request by sebastiankb**
solves #1410

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1500.html" title="Last updated on May 18, 2022, 3:57 PM UTC (75fd278)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1500/78cfc50...75fd278.html" title="Last updated on May 18, 2022, 3:57 PM UTC (75fd278)">Diff</a>

### CODE REVIEW

**Comment by aphillips** [comment]
I see how you got here, but I'm not sure that the addition of the note will be that clear. Note that double-encoding would encode the percent sign (the string wouldn't include any non-ASCII characters--the escaping would process ASCII characters reserved in URL). Perhaps instead:

> Portions of the `tm:ref` value might contain non-ASCII characters that require URL ("percent") encoding before use. Before applying escapes to a `tm:ref` value, implementations should check that the value is not already encoded.

**Comment by sebastiankb** [comment]
@aphillips thanks for your feedback. Text is updated based on you proposal.

File: index.template.html [hunk 1]
```diff
@@ -5141,6 +5141,13 @@ <h3>Extension and Import</h3>
     </span>
     </p>
 
+    <p class="note">
+        Portions of the <code>tm:ref</code> value might contain non-ASCII characters that require URL ("percent") 
+        encoding before use. Before applying escapes to a <code>tm:ref</code> value, implementations should check that the value is not already
```
Comment (egekorkan) [inline index.template.html:5146]: I think we need an RFC assertion here while making sure that the assertion is understandable on its own (implies changes to the sentence)

**Review by egekorkan — CHANGES_REQUESTED** [review]

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge.

@egekorkan also agree to keep it as a note

**Pull request merged**